backend/api/views/main.py

```diff
@@ -33,21 +33,31 @@ def add_<yourclassname>():
     return create_response(status=200, message="success")
 """
 
-# function that is called when you visit /documetns
+# Gets all documents that can be specified using a query string
```

Comment:
Typically function documentation in python uses docstrings, which should be the first lines of a function rather than placed outside of them